internal/infrastructure/rdb/query_mods.go

```diff
@@ -35,5 +36,6 @@ func placeQueryModes(relationsToPlaces ...string) []qm.QueryMod {
 		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlacePhotoAttributions),
 		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlaceReviews),
 		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlaceOpeningPeriods),
+		qm.Load(relation + "." + generated.PlaceRels.PlacePhotos),
```

Comment:
[imo] こちらも、こうなっていると `relation`が`Place`に相当するのでまとまりそうです。
```suggestion
		qm.Load(relation),
		qm.Load(relation + "." + generated.PlaceRels.PlacePhotos),
		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces),
		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlaceTypes),
		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlacePhotoReferences),
		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlacePhotos),
		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlacePhotoAttributions),
		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlaceReviews),
		qm.Load(relation + "." + generated.PlaceRels.GooglePlaces + "." + generated.GooglePlaceRels.GooglePlaceOpeningPeriods),
```